### PR TITLE
Pass command-line arguments 

### PR DESCRIPTION
**Pull request by evalsocket**
# TL;DR
Pass command-line arguments

## Type
 - [x] Bug Fix
 - [ ] Feature
 - [ ] Plugin

## Are all requirements met?

 - [ ] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Tracking Issue
https://github.com/lyft/flyte/issues/167

### CODE REVIEW

**Comment by kumare3** [comment]
I am not sure if we should have strict-mode false - @EngHabu ?

**Comment by evalsocket** [comment]
@EngHabu any update on this PR ?

**Review by EngHabu — DISMISSED** [review]

File: cmd/controller/cmd/root.go [hunk 1]
```diff
@@ -97,11 +97,12 @@ func init() {
 	rootCmd.AddCommand(viper.GetConfigCommand())
 }
 
-func initConfig(_ *cobra.Command, _ []string) error {
+func initConfig(cmd *cobra.Command, _ []string) error {
 	configAccessor = viper.NewAccessor(config.Options{
-		StrictMode:  true,
+		StrictMode:  false,
```
Comment (EngHabu) [inline cmd/controller/cmd/root.go:102]: I don't think this is the complete fix here... I understand it'll allow it to pass... I think the more fundamental fix is to make InitializePFlags keep track of all the flags (and persistent flags) that it sees at the time of initialization. Once it does, it let those flags be skipped while everything else must be recognized (if strictMode is true)...

Comment (kumare3) [reply cmd/controller/cmd/root.go:102]: @EngHabu is the reason for the problem here that, there are other modules registering flags, and or config, will not recognize them?

**Comment by evalsocket** [comment]
@EngHabu Any update on this?....you can test it with my image `evalsocket/flytepropeller:latest`

**Pull request closed without merging**